### PR TITLE
LAT-1144 | Update DateTime format to 3 digits

### PR DESCRIPTION
**Pull request by matthewmorley**

### CODE REVIEW

File: src/Entity/Capture.php [hunk 1]
```diff
@@ -145,8 +145,14 @@ public function setEventSource($eventSource)
...
     {
         // Lift Profile Manager is currently accepting this ISO 8601 format:
-        // "2017-02-07T19:59:53.123456Z".
-        $this['event_date'] = $eventDate->format('Y-m-d\TH:i:s.u\Z');
+        // "2017-02-07T19:59:53.123Z" (or "2017-02-07T19:59:53.000Z" pre 7.0 ).
+        // todo: the above comment needs updating
+        // todo: works for all but hhvm-3.18.5
+        $format = 'Y-m-d\TH:i:s.v\Z';
+        if (version_compare(phpversion(), '7.0', '<')) {
+            $format = 'Y-m-d\TH:i:s.000\Z';
```
Comment (chilic) [inline src/Entity/Capture.php:153]: @matthewmorley What about use something like `->format(DateTime::ISO8601)`?

Comment (matthewmorley) [reply src/Entity/Capture.php:153]: We should discuss with the team(s), I dont know what the actual requirement/desire is around fraction of a second granularity. Sometimes it is useful, even if not a requirement... 

Using DateTime::ISO8601 looks like: https://3v4l.org/MoEpT which will certainly be more predictable.

Comment (mortenson) [reply src/Entity/Capture.php:153]: @matthewmorley My team has no requirements for microsecond granularity. Just seconds would be OK.

**Review by ynx — APPROVED** [review]
@matthewmorley, code looks great. Please merge any time (or I can help with the merge if you want).

**Pull request merged**